packages/web/components/Guild/GuildJoin.tsx

```diff
@@ -105,3 +87,238 @@ export const GuildJoin: React.FC = () => {
     </FlexContainer>
   );
 };
+
+export const JoinCopy: React.FC = () => {
+  const css = {
+    '.header': {
+      marginTop: 5,
+      marginBottom: 1,
+      fontSize: 'xl',
+      color: 'blueLight',
+    },
+    '& br': {
+      marginBottom: 2,
+    },
+    p: {
+      lineHeight: '1.4em',
+    },
+    div: {
+      marginTop: 2,
+      marginBottom: 2,
+    },
+    ul: {
+      marginBottom: 3,
+      marginTop: 1,
+      li: {
+        fontSize: 'sm',
+        ul: {
+          marginBottom: 1,
+        },
+      },
+    },
+  };
+
+  return (
+    <Box sx={css}>
+      <Text>
+        Hello Guilder!
+        <br />
+        Thinking maybe your project should join MetaGame but aren't sure?
+      </Text>
+      <Text className="header">Introduction</Text>
+      <Text>
+        At MetaGame, we believe Ethereum is a core technology for building
+        socioeconomic systems of the future &amp; DAOs are how we get there. We
+        are building an Onboarding Machine &amp; A Decentralized Factory for
+        this fledgling Ethereum DAO ecosystem &amp; the people that want to take
+        part in building the future.
+        <br />
+        Our first goal is to build a place for people to learn what these
+        technologies mean for them &amp; how they can start contributing, then
+        support the people &amp; projects that are building different pieces of
+        this “decentralized society” puzzle in any way we can.
+        <br />
+        We are looking for projects that are either building pieces of the
+        infrastructure for the society of the future, offering tools &amp;
+        services to those that are, or just doing something cool.
+        <Box fontStyle="italic">
+          Please have a clear vision of how you fit in as a piece of this puzzle
+          &amp; ethos before trying to join.
+        </Box>
+        <Box fontWeight="bold">Think:</Box>
+      </Text>
+      <UnorderedList>
+        <ListItem>
+          How does my DAO fit into this “Decentralized Factory”?
+        </ListItem>
+        <ListItem>
+          If it doesn't, does it at least fit as a part of this “building a new
+          world” narrative? (We'll be accepting DAOs ranging from artist or
+          gamer collectives to activist groups.)
+        </ListItem>
+      </UnorderedList>
+      <Text fontWeight="bold">Read more about MetaGame:</Text>
+      <UnorderedList>
+        <ListItem>
+          <MetaLink href="/">MetaGame</MetaLink>
+        </ListItem>
+        <ListItem>
+          <MetaLink
+            isExternal
+            href="https://wiki.metagame.wtf/docs/wtf-is-metagame/wtf-is-metagame"
```

Comment:
maybe you could link them to `/learn/wiki`

Reply:
ah, good idea